cloudflare/resource_cloudflare_worker_kv.go

```diff
@@ -0,0 +1,127 @@
+package cloudflare
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+
+	cloudflare "github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/pkg/errors"
+)
+
+func resourceCloudflareWorkerKV() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceCloudflareWorkersKVCreate,
+		Read:   resourceCloudflareWorkersKVRead,
+		Update: resourceCloudflareWorkersKVUpdate,
+		Delete: resourceCloudflareWorkersKVDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceCloudflareWorkersKVImport,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"key": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"namespace_id": {
+				Type:     schema.TypeString,
+				Required: true,
```

Comment:
I'm wondering if we want a `ForceNew: true` on `namespace_id` to recreate in the event of it changing 🤔

Reply:
That seems reasonable.  I can't think of any downside to doing it.